spec/features/publish/courses/editing_age_range_spec.rb

```diff
@@ -0,0 +1,94 @@
+require "rails_helper"
+
+feature "selecting an age range" do
+  before do
+    given_i_am_authenticated_as_a_provider_user
+    and_there_is_a_course_i_want_to_edit
+  end
+
+  scenario "selecting a preset age range" do
+    when_i_visit_the_edit_age_range_page
+    when_i_select_a_preset_age_range
+    and_i_click_continue
+    then_i_should_see_a_success_message
+    and_the_course_age_range_is_updated("5_to_11")
+  end
+
+  scenario "selecting a custom age range" do
+    when_i_visit_the_edit_age_range_page
+    when_i_select_a_custom_age_range
+    and_i_click_continue
+    then_i_should_see_a_success_message
+    and_the_course_age_range_is_updated("10_to_15")
+  end
+
+  scenario "selecting an invalid age range" do
+    when_i_visit_the_edit_age_range_page
+    when_i_select_an_invalid_age_range
+    and_i_click_continue
+    then_i_should_see_an_error_message
+    and_the_course_age_range_is_not_updated
+  end
+
+private
+
+  def given_i_am_authenticated_as_a_provider_user
+    given_i_am_authenticated(user: create(:user, :with_provider))
+  end
+
+  def and_there_is_a_course_i_want_to_edit
+    given_a_course_exists(enrichments: [build(:course_enrichment, :published)])
```

Comment:
```suggestion
    given_a_course_exists
```

Don't think you need any enrichments